proposals/0000-hybrid-output.md

```diff
@@ -0,0 +1,139 @@
+- Start Date: 2022-10-25
+- Reference Issues: TODO
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+Currently, Astro's [`output`](https://docs.astro.build/en/reference/configuration-reference/#output) enables developers to switch between `'static'` (default) and `'server'` outputs. One of our most requested features is to enable route-level control of `output` so that some routes can be server-rendered while others are statically generated.
+
+# Example
+
+This RFC proposes a new, opt-in `output` mode named `'hybrid'`.
+
+```js
+import { defineConfig } from 'astro/config'
+import node from '@astrojs/node'
+
+export default defineConfig({
+  output: 'hybrid',
```

Comment:
Could we actually leverage this as the default value?  So `output: "server"` will build all pages for SSR - individual pages can opt-out with `export const output = "static"`

Reply:
Yes, this is the direction we're taking in the latest version!